setup.cfg

```diff
@@ -27,6 +27,9 @@ install_requires =
     rtree
     scipy
     utm
+    geopandas
+    shapely
+    networkx
```

Comment:
Wondering if this should be optional

Reply:
I have added these under a new "roadnet" optional deps group. However, now the road net tests are getting skipped. Should I add also these to the "dev" dependencies?